redis/server.cc

```diff
@@ -26,8 +26,8 @@ namespace redis_transport {
 
 static logging::logger logging("redis_server");
 
-redis_server::redis_server(seastar::sharded<redis::query_processor>& qp, auth::service& auth_service, redis_server_config config)
-    : server("Redis", logging)
+redis_server::redis_server(seastar::sharded<redis::query_processor>& qp, auth::service& auth_service, const db::config& cfg, redis_server_config config)
+    : server("Redis", logging, cfg)
```

Comment:
Propagating this config everywhere should have been a separate commit. Now I have to scroll through all this mechanical changes to try to find interesting changes to review.

Reply:
done